dds/Cargo.toml

```diff
@@ -43,3 +43,11 @@ path = "src/implementation/configuration_schema_generator.rs"
 
 [dev-dependencies]
 mockall = { version = "0.11" }
+
+[[bin]]
+name = "subscriber"
+path = "tests/executables/subscriber.rs"

```

Comment:
Can we can these binaries and files something a bit less generic. For example "multiple_subscriber_test" and "publisher_test"?